client/utils/test/account-fees.tsx

```diff
@@ -173,36 +229,36 @@ describe( 'Account fees utility functions', () => {
 			);
 		} );
 
-		it( 'only describes discount if it is different than base fee', () => {
-			const fee = {
-				percentage_rate: 0.123,
-				fixed_rate: 456.78,
-				currency: 'USD',
-			};
-
-			const accountFees = {
-				base: fee,
-				discount: [ fee ],
```

Comment:
After introduction of `BaseFee` and `DiscountFee` types, this test is no longer possible.

Reply:
Restored the test, see 7c4f1bc67373183d73a68114414e54a1f6dd6eb9